.circleci/config.yml

```diff
@@ -1183,13 +1183,31 @@ jobs:
           command: go test -v ./... || true # ignore failures; we're just compiling the tests
       # Run the actual acceptance tests with optimized settings
       - run:
-          name: Run acceptance tests
+          name: Run acceptance tests (devnet=<<parameters.devnet>>, gate=<<parameters.gate>>)
           working_directory: op-acceptance-tests
           no_output_timeout: 60m
           environment:
             GOFLAGS: "-mod=mod"
             GO111MODULE: "on"
           command: just acceptance-test "<<parameters.devnet>>" "<<parameters.gate>>"
+      - run:
+          name: Print results (summary)
+          working_directory: op-acceptance-tests
+          command: |
+            LOG_DIR=$(ls -td -- logs/* | head -1)
+            cat "$LOG_DIR/summary.log" || true
+      - run:
+          name: Print results (failures)
+          working_directory: op-acceptance-tests
+          command: |
+            LOG_DIR=$(ls -td -- logs/* | head -1)
+            cat "$LOG_DIR/failed/*.log" || true
+      - run:
+          name: Print results (all)
+          working_directory: op-acceptance-tests
+          command: |
+            LOG_DIR=$(ls -td -- logs/* | head -1)
+            cat "$LOG_DIR/all.log" || true
```

Comment:
Add `when: always` so that these are executed even if the tests fail (otherwise you'll never see the failures)

https://circleci.com/docs/configuration-reference/#the-when-attribute

Reply:
Or you can make it more fancy and e.g. only run `Print results (failures)` on failure

Reply:
Missed this, thanks!